cishouseholds/edit.py

```diff
@@ -227,14 +228,22 @@ def update_from_csv_lookup(df: DataFrame, csv_filepath: str, id_column: str):
     csv = spark.read.csv(csv_filepath, header=True)
     csv = csv.groupBy("id", "old_value", "new_value").pivot("target_column_name").count()
     cols = csv.columns[3:]
+    window = Window.partitionBy("id").orderBy("id")
```

Comment:
I think the window function could be avoided by creating a `{col}_old` and `{col}_new` for each column to be edited. The the editing loop would just be: use _new if _old matches col, otherwise keep col.
This might be easier to read too, as it's not clear why the `F.last` and `F.row_number` are needed to achieve this

Reply:
the window is required to merge the changes made to the same row together into 1 joinable lookup row